.travis.yml

```diff
@@ -20,6 +20,9 @@ install:
     - source "$HOME/miniconda/etc/profile.d/conda.sh"
     - pip install -U tox-travis
 
+env:
+  global:
+    - ON_TRAVIS=true
 
```

Comment:
Why are you defining a new env variable here? Travis already defines two `TRAVIS` and `CI`. Why not just use those?

Reply:
See  https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/292/commits/a16c0188a644d52be66443f445fd552f067e5341